src/Cubix/Language/Parametric/Derive.hs

```diff
@@ -41,6 +39,17 @@ import Cubix.Sin.Compdata.Derive ( smartFConstructors )
 
 --------------------------------------------------------------------------------
 
+makeIsNotSortInjection :: Name -> Q [Dec]
+makeIsNotSortInjection tName =
+  let t = conT tName
+   in [d| type instance IsSortInjection $t = 'False |]
```

Comment:
The `'` is optional. Just tested.

<img width="649" alt="image" src="https://github.com/user-attachments/assets/0d07eaae-8e0c-4d5c-b1c2-16ea82b9dc5c" />